.changelog/748836b8680843beb820250cd9f01b80.json

```diff
@@ -0,0 +1,8 @@
+{
+    "id": "748836b8-6808-43be-b820-250cd9f01b80",
+    "type": "documentation",
+    "description": "Updated API client and endpoints to latest revision.",
```

Comment:
Description doesn't match the documentation change for `cloudformation`. 

 "description": "Documentation changes.",